Src/signal.c

```diff
@@ -14,6 +14,7 @@
 #include "sounds.h"
 #include "targets.h"
 
+EEprom_t eepromBuffer;
```

Comment:
why is this in signal.c? would main.c make more sense?

Reply:
good question